src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Runtime.Augments;
+using System;
+using System.Reflection;
+
+namespace System
+{
+    public partial class AppDomain
+    {
+        private static AppDomain domain = new AppDomain();
```

Comment:
Or if you would like to be cool, you can write it on single line as: `public static AppDomain CurrentDomain { get; } = new AppDomain();` ;-)